src/libraries/Microsoft.Extensions.Configuration.Binder/tests/Common/ConfigurationBinderTests.Collections.cs

```diff
@@ -2332,6 +2334,40 @@ public void TestMutatingDictionaryValues()
             Assert.Equal("NewValue", dict["Key"][1]);
         }
 
+        [Fact]
+        public void TestCollectionWithNullOrEmptyItems()
+        {
+            string json = @"
+                {
+                    ""CollectionContainer"": [
+                    {
+                        ""Elements"":
+                        {
+                            ""Typdde"": ""UserCredentials"",
+                            ""poop"": ""00"",
```

Comment:
nit: this might trigger policheck
```suggestion
                            ""foo"": ""00"",
```

Reply:
fixed. Thanks for the catch.